lesson_03/homework_03.py

```diff
@@ -16,6 +23,14 @@
 моря становить 37 800 км2. Яку площу займають Чорне та Азов-
 ське моря разом?
 """
+# my answer:
+black_sea_area_km2 = 436402
+azov_sea_area_km2 = 37800
+total_area_km2 = black_sea_area_km2 + azov_sea_area_km2
+black_sea_area_km2 = 436402
+azov_sea_area_km2 = 37800
+total_area_km2 = black_sea_area_km2 + azov_sea_area_km2
```

Comment:
```suggestion
black_sea_area_km2 = 436402
azov_sea_area_km2 = 37800
total_area_km2 = black_sea_area_km2 + azov_sea_area_km2
```
Будьте уважні - дублювання коду варто уникати